cmd/promtool/main.go

```diff
@@ -218,6 +218,7 @@ func main() {
 	analyzeBlockID := tsdbAnalyzeCmd.Arg("block id", "Block to analyze (default is the last block).").String()
 	analyzeLimit := tsdbAnalyzeCmd.Flag("limit", "How many items to show in each list.").Default("20").Int()
 	analyzeRunExtended := tsdbAnalyzeCmd.Flag("extended", "Run extended analysis.").Bool()
+	analyzeMatchers := tsdbAnalyzeCmd.Flag("match", "Series selector to analyze. Only 1 set of matchers is supported now.").String()
```

Comment:
Fairly sure this can just be `.Strings()` and then multiple matches would work?

Reply:
Ah, no a bit more complex, fine to keep as it is, it's just a bit inconsistent with other `--match` options.

Reply:
Yeah it will be a little bit more complex overall. We only supports one single matcher for tsdb dump command as well.
We can have a separate issue to track if we want to have more matchers in the future.

Reply:
Thanks for this.
We're interested in having multiple matchers for `tsdb dump`,  I can create an issue and maybe give it a try.